values.yaml

```diff
@@ -61,6 +61,9 @@ puppetserver:
       ## The LB type (network protocol) for some cloud providers must be set here.
       ports:
         puppetserver:
+          ## WARNING: if a different port is set, PuppetDB and Puppet Compiler/s container images
+          ## must be modified - so that they can connect to Puppet Master/s K8s Service and vice versa.
+          ## Their Docker entrypoint scripts expect port "8140".
```

Comment:
I don't think this is accurate. In the puppetserver container there's the `PUPPET_MASTERPORT` environment variable and in the puppetdb container there's the `PUPPETSERVER_PORT` environment variable so you should be able to change those as long as the env variables are being set

Reply:
Thanks! I wasn't sure neither - therefore, wanted you to check it. 

Will fix it - so changing the ports becomes a working option.